### ISSUE TITLE
FastListTemplate missing css issues

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/panel#4727]
I'm on `panel==1.0.0rc1` trying out the `FastListTemplate` with the `accent` argument.

There are several things that does not work and used to work.

- The switch does not have the white background
- The theme icon does not have the white background
- The plot tooltips does not use the `accent` color
- The bokeh toolbar does not use the `accent` color to indicate selected tools.

![image](https://user-images.githubusercontent.com/42288570/235054064-9003824a-c891-4996-9891-728b9fe16d5c.png)

I think the plot stopped using the `accent` color after a refactor in 0.14.x.

```python



import mimetypes
mimetypes.add_type("application/javascript", ".js")

import hvplot.pandas
import pandas as pd
import numpy as np
import holoviews as hv
import panel as pn

ACCENT = "PURPLE"

pn.extension()

def sine(frequency=1.0, amplitude=1.0, function='sin'):
    xs = np.arange(200)/200*20.0
    ys = amplitude*getattr(np, function)(frequency*xs)
    return pd.DataFrame(dict(y=ys), index=xs).hvplot()

hv_panel = pn.panel(sine)

pn.template.FastListTemplate(
    main=[hv_panel],
    accent=ACCENT,
).servable()
```

I see the below issue in the browser console too.

![image](https://user-images.githubusercontent.com/42288570/235054496-3eb2aa40-1e14-4e23-b346-9859cc5387cc.png)

**Comment by MarcSkovMadsen** [comment]
Ok the header issues is that the Fast javascript does not accept named colors. Must be hex. We should add validation.

```python



import mimetypes
mimetypes.add_type("application/javascript", ".js")

import hvplot.pandas
import pandas as pd
import numpy as np
import holoviews as hv
import panel as pn

ACCENT = "#800080"

pn.extension()

def sine(frequency=1.0, amplitude=1.0, function='sin'):
    xs = np.arange(200)/200*20.0
    ys = amplitude*getattr(np, function)(frequency*xs)
    return pd.DataFrame(dict(y=ys), index=xs).hvplot()

hv_panel = pn.panel(sine)

pn.template.FastListTemplate(
    main=[hv_panel],
    accent=ACCENT,
).servable()
```

![image](https://user-images.githubusercontent.com/42288570/235054784-98ef5212-e538-4b3f-a407-313b95b30576.png)

**Comment by MarcSkovMadsen** [comment]
This is with Panel 0.13.1 and here the plot uses the accent for the bokeh tools and tooltip. Looks much better.

![image](https://user-images.githubusercontent.com/42288570/235225159-31dd428e-daba-4f5b-8c26-9b3ebea7be97.png)

**Comment by MarcSkovMadsen** [comment]
It looks to me like the regression was introduced here https://github.com/holoviz/panel/pull/4275

**Comment by MarcSkovMadsen** [comment]
AS #4275 that introduced the regression was such a big refactor I don't know where to start or end. There is so much CSS that seems to have been dropped. And I don't understand why or how to fix. @philippjfr can you fix or guide me how to fix?

![image](https://user-images.githubusercontent.com/42288570/235230286-7e11415e-e4db-40a6-9392-4a0f92edeb03.png)

**Comment by philippjfr** [comment]
All the component styling code was moved onto the design components, so look in panel/theme/css/fast.css. Everything had to  be adapted to work inside the shadow DOM. Which also means you should be able to use fast design styling in a notebook or in another template. The plot styling definitely seems to have been dropped. Can write up something in some more detail tomorrow.

**Comment by MarcSkovMadsen** [comment]
Related to #4714.

**Comment by MarcSkovMadsen** [comment]
I can also see that the background color of the cards is different. This makes the text harder to read. Especially the blue link is very hard to read.

![image](https://user-images.githubusercontent.com/42288570/235287472-377c4d11-340a-4798-98ac-9ff63ef14186.png)

**Comment by cdeil** [comment]
At https://pyviz-dev.github.io/panel/how_to/styling/design.html  I noticed that when selecting "Fast" the slider isn't displayed:

<img width="369" alt="Screenshot 2023-04-29 at 12 59 34" src="https://user-images.githubusercontent.com/852409/235299151-d5140bc5-fcb5-46c0-8313-97be68856c38.png">

With the other options "Bootstrap", "Material", "Native" it does show up.

**Comment by philippjfr** [comment]
Yes, thanks @cdeil, that has been reported elsewhere. It's not actually an issue with Fast but with the way the docs render the components.

**Comment by philippjfr** [comment]
Recording another item from #4714 here to consolidate the issues:

If you go to https://panel-gallery-dev.pyviz.demo.anaconda.com/portfolio_analyzer 

- [] The reset button in the upper right corner is missing its background color (maybe its on purpose?)
- [] The hyperlink in the table is not using the accent color

![image](https://user-images.githubusercontent.com/42288570/234858196-6ed1b0e3-3f17-4eeb-859f-6614be4f050f.png)

**Comment by philippjfr** [comment]
And another from https://github.com/holoviz/panel/issues/4668:

If you go to the `dev` VideoStream example https://panel-gallery-dev.pyviz.demo.anaconda.com/streaming_videostream, you can see the ButtonGroup is not styled

![image](https://user-images.githubusercontent.com/42288570/233148342-405cda52-c742-4f96-a4d5-cf5adcae1da1.png)

**Comment by philippjfr** [comment]
A few tweaks in https://github.com/holoviz/panel/pull/4793

<img width="314" alt="Screen Shot 2023-05-06 at 12 55 42" src="https://user-images.githubusercontent.com/1550771/236620051-95fb8e16-eeb7-42d6-84ff-7dad193dd7f0.png">

<img width="1245" alt="Screen Shot 2023-05-06 at 12 58 45" src="https://user-images.githubusercontent.com/1550771/236620175-aa19f42e-675d-4bdc-8a15-04c796051a9a.png">

**Comment by philippjfr** [comment]
This issue has gone stale. Any missing CSS issues should be raised in a new issue (or better yet PR).

**Comment by philippjfr** [comment]
Actually this is still a good list, let's keep it open.
